### PR TITLE
dotCMS/core#25425 Add a wall container to show misconfiguration of Analytics app

### LINKED ISSUE #25425: [UI] Show a no `Analytics App not Properly Configured` page
### Parent Issue

_No response_

### User Story

As a user, I want to know early as possible if the experiment App is properly configurated.

### Acceptance Criteria

- Create a Guard to precheck if I can go to Experiment's List
- If the `Analytics App` is not properly configurated show an Empty page with the final design and information
- Validate if the app is properly configurated from `/v1/experiments/_healthcheck` response

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

Screenshot 2023-06-27 at 12 10 52 PM (1)

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* A Guard to pre-check if I can go to Experiment List.

### Checklist
- [x] Tests
- [x] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Screenshot 2023-07-17 at 3 22 46 PM

### <samp>🤖 Generated by Copilot at 6b92bc0</samp>

This pull request adds a new feature flag for loading frontend experiments in the dotCMS platform. It also adds a new route, component, and guard to handle the case when the experiments infrastructure is not working and show a message to the user. Additionally, it adds a reusable component to display a message with an icon, subtitle, button, and contact link. Finally, it adds some translation properties and some minor changes for debugging and formatting purposes. The changes affect the following files: `dot-experiments.service.ts`, `dot-experiments-portlet-routes.ts`, `app-routing.module.ts`, `dot-edit-page-routing.module.ts`, `dot-experiments-analytic-app-misconfiguration.component.html`, `dot-experiments-analytic-app-misconfiguration.component.scss`, `dot-experiments-analytic-app-misconfiguration.component.spec.ts`, `dot-experiments-analytic-app-misconfiguration.component.ts`, `dot-experiments-analytic-app.guard.ts`, `index.ts`, `dot-empty-container.component.html`, `dot-empty-container.component.scss`, `dot-empty-container.component.spec.ts`, `dot-empty-container.component.ts`, and `Language.properties`.

## Related Issue
Fixes #25425 

# Explanation of Changes

### <samp>🤖 Generated by Copilot at 6b92bc0</samp>

*  Add a new feature flag for loading frontend experiments
  * Enable router tracing for debugging purposes ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-6c7f381c5033c96909090cb6d24c4507c999ed3ccec1ec3b74958152f21a572fL208-R209))
  * Add blank lines for readability ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-e085106ab179853e265adf11699afca1c0e2e8a51ea0a89fc3b1755bc25276cfL24-R27))
  * Add `of` operator for mocking observables ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-828cef4c77199001a63f353a6f729a68a3626eb65e103aa2b9d7525d3ae2cb8bL1-R1))
  * Add `healthCheck` method to check experiments infrastructure status ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-828cef4c77199001a63f353a6f729a68a3626eb65e103aa2b9d7525d3ae2cb8bL29-R39))
    * Export component from `ui` library ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-bd390158a0315505d598e7ff896c73589c38f5263e0ff170662313807c45e15eR12))
    * Add translation keys to `Language.properties` ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-21db9723fa4c52389731b7575f1e19bd364aa992b7e549e22c08a33456994e94L5369-R5374))
    * Add translation keys to `Language.properties` ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-21db9723fa4c52389731b7575f1e19bd364aa992b7e549e22c08a33456994e94L5369-R5374))
  * Add `analyticsAppGuard` function to redirect to misconfiguration route if experiments infrastructure is not working ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-ef06edb9fe93ba6ee63d055e0fc67807f6490ae0b361bdf3bd2e482c2fc03587R1-R25))
  * Add new route for misconfiguration component ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-41fcb0f6c755390384152acc4102d9924e869f6bc0cbd90f8fb7555d00da1081L6-R20))
  * Add `canActivateChild` property to page route with `analyticsAppGuard` ([link](https://github.com/dotCMS/core/pull/25531/files?diff=unified&w=0#diff-41fcb0f6c755390384152acc4102d9924e869f6bc0cbd90f8fb7555d00da1081R28-R29))

### CODE REVIEW

**Review by zJaaal — APPROVED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@use "variables" as *;
+
+:host {
+    width: 100%;
+    min-height: 100%;
+    background: $white;
+    display: flex;
+    flex-direction: column;
+    padding-right: 80px;
```
[the remark this answers is not in the document]
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.scss:9]: The nav is `80px` width, I don't know if convert this to Rem. Later we will change from 14px to 16px and we need to fix then that.

**Review by hmoreras — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Review by manuelrojas — APPROVED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmontes) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.scss:9]: True

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    [title]="'experiments.analytics-app-misconfiguration.route-title' | dm"
+    (goBack)="goToBrowserBack()"
+/>
+<div class="flex justify-content-center align-items-center h-full">
+    <dot-empty-container
+        [buttonLabel]="'experiments.analytics-app-misconfiguration.btn-label' | dm"
+        [icon]="'pi-chart-line'"
+        [subtitle]="'experiments.analytics-app-misconfiguration.subtitle' | dm"
+        [title]="'experiments.analytics-app-misconfiguration.title' | dm"
+        (buttonAction)="goToAnalyticsApp()"
```
[the remark this answers is not in the document]
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.html:11]: Yes, but it is less explicit. I will group together all of the `basic` block and the `extra` block (Button and link of go to contact us) as Input?

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        component: DotExperimentsAnalyticAppMisconfigurationComponent,
+        providers: [mockProvider(DotMessageService)]
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+
+    it('should DotEmptyContainerComponent exist in the component', () => {
+        expect(spectator.query(DotEmptyContainerComponent)).toExist();
```
Comment (fmontes) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts:20]: Throw a comment about the integration then.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts (hunk 3 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts:27]: Added 2 small test.

Comment (fmontes) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts:27]: I mean add a comment here to tell that this is being tested in other places.

**Review by fmontes — APPROVED** [review]

**Pull request merged**